### DISCUSSION TITLE
Two features for smoother workflow when creating notes

### DISCUSSION THREAD

**Discussion by ludberg** [standardnotes/forum#3343] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I create new notes (PC) that I have to use my mouse to click on the auto-generated note name in order to rename the note. 
Also, I'm frustrated that I can't hit the tab key to easily jump between the note name and content. 

**Describe the solution you'd like**
When I create a new note, I would like the focus to be on the note name in edit mode.

When I'm done with naming my note, I would like to be able to hit the tab key and jump straight to the content. As of today, I have to press it several times. 

**Describe alternatives you've considered**
Perhaps a keyboard shortcut for "rename note"

**Additional context**

**Comment by munkmode** [comment]
+1
